central/compliance/service/service_impl.go

```diff
@@ -144,6 +154,14 @@ func (s *serviceImpl) GetRunResults(ctx context.Context, request *v1.GetComplian
 	if err != nil {
 		return nil, err
 	}
+	config, exist, errConfig := s.complianceDataStore.GetConfig(ctx, request.GetStandardId())
+	if errConfig != nil {
+		return nil, errConfig
+	}
+	if exist && config.GetHideScanResults() {
+		return nil, errors.New("Results are hidden for this standard")
```

Comment:
not: `errors.New("Results are hidden for this standard")` to `errors.Errorf("Results are hidden for standard with ID %s", request.GetStandardID())`

Reply:
Did we decide on this? I think that it makes sense to let users still see the results